src/main/resources/assets/chisel/lang/de_DE.lang

```diff
@@ -23,7 +23,7 @@ chisel.tooltip.blockname=%1$s (%2$s)
 tile.chisel.blockAluminum.name=Aluminiumblock
 tile.chisel.blockAluminum.caution.desc.1=Vorsicht
 tile.chisel.blockAluminum.crate.desc.1=Versandkiste
-tile.chisel.blockAluminum.thermal.desc.1=Warm
+tile.chisel.blockAluminum.thermal.desc.1=Glühend
```

Comment:
didn't knew that, but that said - wouldn't it be better to keep it as "Thermal" to slightly refer to the mod? "Warm" or "Glühend" does not make a difference unter these circumstances I think, it just sounds better in my opinion.